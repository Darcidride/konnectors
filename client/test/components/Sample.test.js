/* global test, expect */
'use strict'

import React from 'react'
import renderer from 'react-test-renderer'

const Dummy = () => <p>Hello world</p>

test('Hello world', () => {
  const component = renderer.create(
    <Dummy />
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
