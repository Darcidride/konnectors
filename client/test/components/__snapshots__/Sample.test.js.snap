exports[`test Hello world 1`] = `
<p>
  Hello world
</p>
`;
